core/plugins/plugins.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <unordered_map>
+#include "NvInferPlugin.h"
+#include "NvInferPluginUtils.h"
+#include "core/util/prelude.h"
+
+namespace trtorch {
+namespace core {
+namespace plugins {
+namespace impl {
+// Helper class which registers a plugin in trtorch namespace
+template <typename T>
+class PluginRegistrar {
+ public:
+  PluginRegistrar() {
+    getPluginRegistry()->registerCreator(instance, "trtorch");

```

Comment:
add nvinfer1:: here to be clear

Reply:
Just saw `getPluginRegistry` is not in any namespace.